Makefile

```diff
@@ -114,6 +114,9 @@ install-completions:
 shell: build-container
 	$(DOCKER_RUN_DOCKER) bash
 
+vendor-image:
+	vndr -whitelist '^github.com/containers/image/docs/.*' github.com/containers/image
```

Comment:
Overall, `README.md` needs updating as well.

I’m worried having a special command for vendoring c/image (as opposed to anything else) will get overlooked, though.

Maybe
```make
vendor: vendor.conf
	vndr -whitelist '^github.com/containers/image/docs/.*'
```
and then document `make vendor`?  OTOH PRs frequently update only the c/image subpackage (people understandably don’t want to be on the hook for changes in other packages they don’t care for).

I don’t know what’s the usual practice.

Reply:
How about we add a test that will fail if the docs are not there and report an error explaining that you should execute make vendor if you are changing containers/image.

Reply:
I updated the README.md, also added test to check for docs.